deployment/ccip/changeset/v1_5_1/cs_deploy_ping_pong.go

```diff
@@ -0,0 +1,198 @@
+package v1_5_1
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset"
+
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/latest/ping_pong_demo"
+)
+
+var _ deployment.ChangeSet[DeployPingPongDemoContractsConfig] = DeployPingPongDemoContractsChangeset
+var _ deployment.ChangeSet[StartPingPongDemoContractsConfig] = StartPingPongDemoContractsChangeset
+var _ deployment.ChangeSet[SetPausedPingPongDemoContractsConfig] = SetPausedPingPongDemoContractsChangeset
+var _ deployment.ChangeSet[SetConterpartPingPongDemoContractsConfig] = SetCounterpartPingPongDemoContractsChangeset
+
```

Comment:
nit : This is legacy way of writing changesets- we are now following - this - https://github.com/smartcontractkit/chainlink/blob/develop/deployment/ccip/changeset/v1_5_1/cs_add_token_e2e.go#L52